change/@ni-nimble-components-decb31d7-985c-4f99-909a-20315ad0fc1a.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "none",
+  "comment": "Enabling Chromatic TurboSnap to reduce the number of Chromatic snapshots consumed by the Nimble build pipeline.",
+  "packageName": "@ni/nimble-components",
+  "email": "1458528+fredvisser@users.noreply.github.com",
+  "dependentChangeType": "none"
```

We are adding to the component public api now

```suggestion
  "type": "minor",
  "comment": "Enabling Chromatic TurboSnap to reduce the number of Chromatic snapshots consumed by the Nimble build pipeline.",
  "packageName": "@ni/nimble-components",
  "email": "1458528+fredvisser@users.noreply.github.com",
  "dependentChangeType": "minor"
```

---

👀 @rajsite How do you make a multi-line suggestion?

---

Lol, it was definitely something I looked up, doesn't seem unintuitive, but never got it without docs https://github.blog/changelog/2020-04-15-multi-line-code-suggestions-general-availability/